templates/rhsm-katello-reconfigure.erb

```diff
@@ -31,6 +31,17 @@ then
 
   # configure rhsm
   # the config command was introduced in rhsm 0.96.6
+  subscription-manager config \
+    --server.hostname="$KATELLO_SERVER" \
+    --server.prefix="$PREFIX" \
+    --server.port="$PORT" \
+    --rhsm.repo_ca_cert="%(ca_cert_dir)s$KATELLO_SERVER_CA_CERT" \
+    --rhsm.baseurl="$BASEURL"
+elif uname -v | grep -qi debian
```

Comment:
Can we use /etc/os-release or a fact for this? Though I don't want to add another dependency.

Reply:
ATM we transform the built RPM into a DEB. That way we can install and run the script.